doc/RELEASE_NOTES.md

```diff
@@ -29,49 +31,50 @@ Note that work on the `v0.71` release started in earnest with the `v0.69.3-agent
+While we don't include installation instructions for the new `node-exporter` and `dcgm` tools in the published documentation, you can find a manual installation procedure for the Prometheus "node_exporter" and references to the Nvidia "DCGM" documentation in the [`agent/tool-scripts/README`](https://github.com/distributed-system-analysis/pbench/blob/v0.71.0-beta.0/agent/tool-scripts/README.md).
 
 Container images built using the above RPMs are available in the [Pbench](https://quay.io/organization/pbench) organization in the Quay.io container image repository using tags `beta`, `v0.71.0-XX`, and `XXXXXXXXX`.
 
 
 Summary of Changes
 ====
 
-## Support for RHEL 9 & CentOS 9
+## Support for RHEL 9 & CentOS Stream 9
 
-Support for RHEL & CentOS 9 is provided in this release.  Note that since RHEL 9 has not been GA'd yet there might still be some changes that will have to be made to support it.
+Support for RHEL & CentOS Stream 9 is provided in this release.
 
 
 ## The New "Tool Meister" Sub-System
 
 The "Tool Meister" sub-system (introduced by PR #1248) is the major piece of functionality delivered with the release of `v0.71` of the pbench-agent.
 
-This is a significant change, where the pbench-agent first orchestrates the instantiation of a "Tool Meister" process on all hosts registered with tools, using a Redis instance to coordinate their operation, and the new "Tool Data Sink" process handles the collection of data into the pbench run directory hierarchy.  This effectively eliminates all remote SSH operations for individual tools except one per host to orchestrate the creation of the Tool Meister instance.
+This is a significant change, where the pbench-agent first orchestrates the instantiation of a "Tool Meister" process on all hosts registered with tools, using a Redis instance to coordinate their operation, and the new "Tool Data Sink" process handles the collection of data into the pbench run directory hierarchy.  This effectively eliminates all remote SSH operations for individual tools except the initial one per host to create each Tool Meister instance.
 
-One Tool Meister instance is created per registered host, and then a single Tool Data Sink instance is created on the host where the benchmark script is run.  The Tool Data Sink is responsible for collecting and storing locally all data sent to it from the deployed Tool Meister instances.
+One Tool Meister instance is created per registered host, and then a single Tool Data Sink instance is created on the host where the benchmark convenience script is run.  The Tool Meister instances are responsible for running the registered tools for that host, collecting the data generated as appropriate. The Tool Data Sink is responsible for collecting and storing locally all data sent to it from the deployed Tool Meister instances.
```

Comment:
s/for that/on their respective/

Reply:
Do you want to say anything about the TM and the localhost case?

Reply:
I am not sure it is worth talking about that detail.